export const COMMON_ERRORS: Record<string, string> = {
  '424': 'Erro inesperado em uma dependÃªncia externa.',
  '429': 'Muitas tentativas. Tente novamente mais tarde.',
  '500': 'Erro interno. Por favor, tente novamente.',
};
